packages/codemods/src/tests/core-imports.test.ts

```diff
@@ -45,9 +45,8 @@ return something()
   it('should refactor imports', () => {
     const input = `
 /* COPYRIGHT: ME */
-import { Button as MyButton, createApiRef, createApp } from '@backstage/core';
```

Comment:
I'd also like to bring this change to your attention. As `createApp` is no longer found in `core-app-api` this unit test began to fail. I suspect removing it like I did may not be the most ideal approach -- suggestions are welcome.

Reply:
You could use `createSpecializedApp` for example instead. The thing it's testing is to make sure that an import that was moved to `core-app-api` is migrated correctly, so anything from `core-app-api` will do. It does open up the question of whether we should update the codemod to import from app-defaults instead, but I think we could at least leave that for later.